omnigibson/simulator.py

```diff
@@ -501,6 +493,10 @@ def play(self):
             # We need to do this because for some reason omniverse exhibits strange behavior if we do certain operations
             # immediately after playing; e.g.: syncing USD poses when flatcache is enabled
             if was_stopped:
+                if self.scene is not None and self.scene.initialized:
+                    for robot in self.scene.robots:
+                        robot.update_controller_mode()
```

Comment:
can we add a comment here why this is needed? (feel free to add a todo comment here also since I'm following up on this)

Reply:
done